MANUAL_REGRESSION.md

```diff
@@ -374,6 +384,29 @@ On iOS:
     - user should see the option to send SMS
     - the SMS input flag should be the UK one
 
+## Internal - functional
+##### 1. Prevent upload fallback when requested
+
+0. Given user opened the link with `uploadFallback=false` flag
+1. And user is on first page of cross-device flow
+2. Open the cross device link on mobile browsers with a working webcam.
+    - user should be able to complete the cross-device flow successfully.
+
+0. Given user opened the link with `uploadFallback=false` flag
+1. And user is on first page of cross-device flow
+2. Open the cross device link on mobile browsers with a malfunctioning webcam.
```

Comment:
Maybe worth changing this step to `2. Open the cross device link on mobile browsers with a malfunctioning webcam or on mobile browsers that do not support getUserMedia (ie Safari on iOS10.3 or earlier).`

Reply:
done :)